solr/core/src/java/org/apache/solr/handler/export/ExportWriter.java

```diff
@@ -251,13 +260,13 @@ protected int transferBatchToArrayForOutput(SortQueue queue, SortDoc[] destinati
         destinationArr[++outDocsIndex] = s;
       }
     }
-
     return outDocsIndex;
   }
 
-  protected void addDocsToItemWriter(List<LeafReaderContext> leaves, IteratorWriter.ItemWriter writer, SortDoc[] docsToExport, int outDocsIndex) throws IOException {
+  protected void addDocsToItemWriter(List<LeafReaderContext> leaves, IteratorWriter.ItemWriter writer, SortDoc[] docsToExport, int outDocsIndex, int numDocsToExclude) throws IOException {
```

Comment:
you are changing the interface here, are you sure you really really need to do that?